packages/react/src/auto/shadcn/ShadcnAutoForm.tsx

```diff
@@ -39,29 +41,27 @@ export const makeAutoForm = <Elements extends ShadcnElements>(elements: Elements
 
   const AutoSubmit = makeShadcnAutoSubmit(elements);
   const { SubmitSuccessfulBanner, SubmitErrorBanner, SubmitResultBanner } = makeSubmitResultBanner(elements);
-
-  const { Form, Skeleton } = elements;
-
   const AutoHasOneForm = makeShadcnAutoHasOneForm(elements);
   const AutoBelongsToForm = makeShadcnAutoBelongsToForm(elements);
   const AutoHasManyForm = makeShadcnAutoHasManyForm(elements);
   const AutoHasManyThroughForm = makeShadcnAutoHasManyThroughForm(elements);
+
+  const FormContainer = forwardRef<HTMLFormElement, React.FormHTMLAttributes<HTMLFormElement>>(({ className, ...props }, ref) => {
```

Comment:
FYI @jadewale -- I had to switch us away from using the `Form` component from shadcn, as it messes with the Autoform context because it also uses `react-hook-form` by default. The version we were using for testing in this repo was edited to not do that, but we can't expect our users to make the same edits, so I just switched to a hardcoded container here instead of accepting a form element.